### ISSUE TITLE
Feature request: Add support for ViSQOL

### ISSUE THREAD

**Issue by iver56** [aliutkus/speechmetrics#15]
Thanks for making speechmetrics!

Here's a candidate implementation of ViSQOL: https://github.com/google/visqol

**Comment by aliutkus** [comment]
I'm open to a PR

**Comment by iver56** [comment]
I agree, integration with Python seems non-trivial. I added a feature request for Python API here: https://github.com/google/visqol/issues/3

**Comment by JohnHerry** [comment]
> I agree, integration with Python seems non-trivial. I added a feature request for Python API here: [google/visqol#3](https://github.com/google/visqol/issues/3)

It just add a python setup.py file. but without a direct usable binary. the bazel compiler is horrable.

**Comment by JohnHerry** [comment]
The facebook Noresqa is more friendly to pythoners, and easy to integrade.
